internal/envoy/install.go

```diff
@@ -75,9 +79,68 @@ func InstallIfNeeded(ctx context.Context, o *globals.GlobalOpts, v version.Versi
 		// TODO: figure out how to get a stat error that isn't file not exist so we can test this
 		return "", err
 	}
+
+	// Set the inferred Envoy version here, hence it can be used in the subsequent steps.
+	o.EnvoyVersion = v
```

Comment:
mention which subsequent steps need this? I hope we don't have any and you can delete this. The main reason for the version is just to get the install path, which this method returns. If you delete this, what breaks?